deparse.c

```diff
@@ -27,9 +27,11 @@
 #endif
 #include "commands/defrem.h"
 #include "mb/pg_wchar.h"
+#include "nodes/makefuncs.h"
 #include "nodes/nodeFuncs.h"
 #include "optimizer/tlist.h"
 #include "parser/parsetree.h"
+#include "parser/parse_type.h"
 #include "utils/builtins.h"
 #include "utils/lsyscache.h"
 #include "utils/syscache.h"
```

Comment:
This comment does not relate to the change in this PR, but could you fix the typo error `dosens't` at line 2791?

Reply:
Fixed, @son-phamngoc. please verify.

Reply:
Thank you. Confirmed. Please help me to resolve this comment.